elytron/src/main/java/org/wildfly/extension/elytron/FileSystemRealmDefinition.java

```diff
@@ -139,6 +199,25 @@ private RealmAddHandler() {
             super(SECURITY_REALM_RUNTIME_CAPABILITY, ATTRIBUTES);
         }
 
+        private SecretKey getSecretKey(OperationContext context, String credentialStoreReference, String alias) throws OperationFailedException {
```

Comment:
Looks like this could be made `private static`.

Reply:
Changed